modules/core/src/test/java/org/apache/ignite/internal/processors/datastreamer/DataStreamerImplSelfTest.java

```diff
@@ -149,8 +151,10 @@ public void testNullPointerExceptionUponDataStreamerClosing() throws Exception {
             dataLdr.close(true);
         }
         catch (CacheException | IllegalStateException ignore) {
-            // This is ok to ignore this exception as test is racy by it's nature -
-            // grid is stopping in different thread.
+            failed = true;
```

Comment:
You should create a separate test, because this test is about closing streamer during stop grids and exception here may be caused by stopping grid.